src/js/Edit/L.PM.Edit.js

```diff
@@ -14,6 +14,10 @@ const Edit = L.Class.extend({
     limitMarkersToCount: -1,
     hideMiddleMarkers: false,
     draggable: true,
+    allowEditing: true, //TODO: should it disable all interactions on a layer if `enable()` is called? For example a Circle can also be dragged -> But it is in Edit-Mode
+    allowDragging: true, //Todo: same as draggable? -> Not direct but can confuse the user. Draggable are only options for Marker to make it draggable (Leaflet Core)
```

Comment:
use allowDragging and make draggable deprecated -> draggable of Leaflet will be overwritten

Reply:
I suggest to do this in the next major release